custom_functions/fast_etl.py

```diff
@@ -1170,3 +1171,32 @@ def search_key_gaps(
                     delta_time = time.perf_counter() - start_time
                     print('Tempo do compare: {:.4f} segundos'.format(delta_time))
                     print("Resumo: ", gaps, "gaps, ", totdif, "rows faltam no destino!")
+
+def load_env_var(
+    conn_name: str,
+    conn_id: str):
+    """
+    # Atribui a string de conexão para uma variável de ambiente em execução
+    # Útil para executar o Great Expectations através de uma DAG, utilizando
+    # as credenciais das connections do próprio airflow e referenciando
+    # nas variaveis setadas no arquivo great_expectations.yml
+
```

Comment:
Sugiro deixar sem os "`#`" no início da linha de texto.

```suggestion
    Atribui a string de conexão para uma variável de ambiente em execução.
    Útil para executar o Great Expectations através de uma DAG, utilizando
    as credenciais das connections do próprio airflow e referenciando
    nas variáveis setadas no arquivo `great_expectations.yml`.

```